migrations/Version202608031745002141_taoItems.php

```
@@ -0,0 +1,55 @@
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
+
+        $this->addReport(
+            Report::createSuccess(
+                'Item Comment ontology synced; services are wired via ItemCommentServiceProvider'
+            )
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)